tests/integration/test_installation.py

```diff
@@ -16,37 +14,7 @@
 from databricks.labs.ucx.workspace_access.groups import GroupManager
 
 logger = logging.getLogger(__name__)
-
-
-def test_destroying_non_existing_schema_fails_with_correct_message(ws, sql_backend, env_or_skip, make_random):
```

Comment:
You cannot just remove the test :)   It verifies that an error message is propagated correctly, otherwise it's extremely difficult to debug test failures.